src/components/z-offcanvas/index.stories.mdx

```diff
@@ -55,61 +75,10 @@ import "./index";
               massa. Nulla facilisi. Ut id quam at ante sagittis posuere vitae sit amet sem. Fusce suscipit ante
               pellentesque felis facilisis tincidunt.
             </span>
-          </div>
-        </z-offcanvas>
-      </div>
-    `}
+        </div>
+      </z-offcanvas>
+    </div>`}
   </Story>
-</Preview>
+</Canvas>
 
-<Preview>
```

Comment:
personalmente questa l'avrei lasciata in modo da avere le due schede che dimostrano il corretto comportamento di entrata in entrambi i casi del pushcontent (entrata da sx o da dx), altrimenti in uno dei due il componente è come se apparisse dal nulla in mezzo alla pagina e sembra un bug